eslint.config.ts

```diff
@@ -0,0 +1,22 @@
+import tsParser from "@typescript-eslint/parser";
```

Comment:
this file should be a `.js` i believe

Reply:
Yes, that is right. Actually it can be other formats as well, but I found out `ts` is not accepted.